src/app/checkout/Checkout.tsx

```diff
@@ -179,7 +180,16 @@ class Checkout extends Component<CheckoutProps & WithCheckoutProps & WithLanguag
                 this.handleReady();
             }
         } catch (error) {
-            this.handleUnhandledError(error);
+            if ( error.status === 500 && error.message === 'Tax Service Unavailable' ) {
```

Comment:
Assuming we can identify `Tax Service Unavailable` with these.

Reply:
Is `502` a better fit here? Are there any other similar circumstances that we handle in the checkout context? If so, what HTTP error codes are expected there? e.g. payment provider is down.

Reply:
We will work on the error code and message when the backend work begins. 👨🏻‍💻